composer.json

```diff
@@ -42,6 +42,8 @@
             "src/"
         ]
     },
+    "minimum-stability": "dev",
```

Comment:
your link there answers my question with, no, it's not needed, and shouldn't be added for no reason